owncloudData/src/main/java/com/owncloud/android/data/authentication/datasources/implementation/OCLocalAuthenticationDataSource.kt

```diff
@@ -65,8 +67,14 @@ class OCLocalAuthenticationDataSource(
             userName = userName,
             password = password,
             updateAccountWithUsername = updateAccountWithUsername
-        ).also {
-            updateUserAndServerInfo(it, serverInfo, userInfo)
+        ).also { account ->
+            updateAccountWithUsername?.let {
+                accountManager.setPassword(account, password)
+                SingleSessionManager.getDefaultSingleton().refreshCredentialsForAccount(
```

Comment:
Just curious, why don't you need this for OAuth?

Reply:
Applied for OAuth too 👍